library/core/src/num/error.rs

```diff
@@ -74,26 +74,20 @@ pub struct ParseIntError {
 /// # Example
 ///
 /// ```
-/// #![feature(int_error_matching)]
-///
 /// # fn main() {
 /// if let Err(e) = i32::from_str_radix("a12", 10) {
 ///     println!("Failed conversion to i32: {:?}", e.kind());
 /// }
 /// # }
 /// ```
-#[unstable(
-    feature = "int_error_matching",
-    reason = "it can be useful to match errors when making error messages \
-              for integer parsing",
-    issue = "22639"
-)]
+#[stable(feature = "int_error_matching", since = "1.54.0")]
 #[derive(Debug, Clone, PartialEq, Eq)]
 #[non_exhaustive]
 pub enum IntErrorKind {
     /// Value being parsed is empty.
     ///
     /// Among other causes, this variant will be constructed when parsing an empty string.
+    #[stable(feature = "int_error_matching", since = "1.54.0")]
     Empty,
```

Comment:
> Among other causes, this variant will be constructed when parsing an empty string.

This confused me -- what are the other causes other than an empty input string where the caller might get back `IntErrorKind::Empty`?

Reply:
Arguably "+" on its own might be classed as empty?

Reply:
I think that line may be a relic of some of the old behaviour that was changed in #77640

I'll verify that as soon as I get the chance (life has been busy) and probably remove that line.